Index: backend/examen/src/main/java/com/k3/examen/config/SwaggerConfig.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/backend/examen/src/main/java/com/k3/examen/config/SwaggerConfig.java b/backend/examen/src/main/java/com/k3/examen/config/SwaggerConfig.java
new file mode 100644
--- /dev/null	(date 1777747119394)
+++ b/backend/examen/src/main/java/com/k3/examen/config/SwaggerConfig.java	(date 1777747119394)
@@ -0,0 +1,24 @@
+package com.k3.examen.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class SwaggerConfig {
+    @Bean
+    public OpenAPI openAPI() {
+        return new OpenAPI()
+                .info(new ProcessHandle.Info()
+                        .title("Event Planner API")
+                        .description("Documentation de l'API Event Planner")
+                        .version("1.0.0"))
+                .addSecurityItem(new SecurityRequirement()
+                        .addList("Bearer Auth"))
+                .components(new Components()
+                        .addSecuritySchemes("Bearer Auth", new SecurityScheme()
+                                .type(SecurityScheme.Type.HTTP)
+                                .scheme("bearer")
+                                .bearerFormat("JWT")));
+    }
+
+}
